conf/messages.en

```diff
@@ -323,28 +323,48 @@ expenses.no = No
 ## Office supplies
 officeSupplies.title = Office supplies
 officeSupplies.heading = Office supplies
-officeSupplies.esAllowable = Yes, all of my expenses are allowable
-officeSupplies.yesDisallowable = Yes, some of my expenses are disallowable
+officeSupplies.l1 = phones, including mobiles
+officeSupplies.l2 = internet, email and fax running costs
+officeSupplies.l3 = postage, stationery and printing
+officeSupplies.l4 = small office equipment
+officeSupplies.l5 = computer software costs
+officeSupplies.l6 = equipment and computer hardware
+officeSupplies.l7 = any amounts that cover personal use
+officeSupplies.l8 = new phones or computer hardware
+officeSupplies.question.individual = Did you buy any office supplies?
+officeSupplies.question.agent = Did your client buy any office supplies?
 officeSupplies.checkYourAnswersLabel = Office supplies
-officeSupplies.error.required = Select officeSupplies
+officeSupplies.error.required.individual = Select if all your office supplies expenses are allowable, some of them are disallowable or you do not have any office supplies expenses
+officeSupplies.error.required.agent = Select if all your client’s office supplies expenses are allowable, some of them are disallowable or they do not have any office supplies expenses
 officeSupplies.change.hidden = OfficeSupplies
 
 ## Taxi Minicab or Road Haulage
 taxiMinicabOrRoadHaulage.title = Were you a taxi, minicab or road haulage industry driver?
-taxiMinicabOrRoadHaulage.heading = Were you a taxi, minicab or road haulage industry driver?
-taxiMinicabOrRoadHaulage.yes = Yes
-taxiMinicabOrRoadHaulage.no = No
+taxiMinicabOrRoadHaulage.heading.individual = Were you a taxi, minicab or road haulage industry driver?
```

Comment:
Should we remove heading messages and instead use title as page title and heading, like we've done on the other pages. Means the title will be UserType specific as well